### PR TITLE
Important Minor Text Fix

### PR DESCRIPTION
**Pull request by BrownSlaughter**
Important Minor Text Fix

## Description

Doing the needful to an Important file

### CODE REVIEW

**Comment by tomballgithub** [comment]
For what attempts to be a professional project, this example.py file is not so.  We should delete it altogether

File: example.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 print("Please refer to documentation, there is no more example.py, " +
       "only runserver.py")
-fuck = raw_input("Would yous till like to massage the devs that " +
```
Comment (Arengo) [inline example.py]: Sorry, but you're clearly mistaken with the second part of this string. So, who's up here to ``massage`` me?

Comment (Obihoernchen) [inline example.py]: Oh noooo revert this :o !!!!!!!

**Comment by BrownSlaughter** [comment]
Bluemode stole my pr, I am rage closing this PR and demanding example.py is deleted :)

**Comment by bluemode** [comment]
Yeah, I am awful

**Pull request closed without merging**